air/src/execution_step/boxed_value/jvaluable.rs

```diff
@@ -18,6 +18,7 @@ mod canon_stream;
 mod cell_vec_resolved_call_result;
 mod iterable_item;
 mod resolved_call_result;
+mod stream_map;
```

Comment:
We don't need `JValuable` implementation thanks to https://github.com/fluencelabs/aquavm/pull/576